src/applications/edu-benefits/tests/wizard/containers/WizardContainer.unit.spec.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import WizardContainer from '../../../wizard/containers/WizardContainer';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import {
+  getReferredBenefit,
+  getWizardStatus,
+  WIZARD_STATUS_COMPLETE,
+  NO_BENEFIT_REFERRED,
+} from '../../../../static-pages/wizard';
+
+describe('WizardContainer', () => {
+  const mockStore = {
+    sessionStorage: {},
+  };
+  let setWizardStatus;
+
+  before(() => {
+    global.sessionStorage = {
+      getItem: key =>
+        key in mockStore.sessionStorage ? mockStore.sessionStorage[key] : null,
+      setItem: (key, value) => {
+        mockStore.sessionStorage[key] = `${value}`;
```

Comment:
`mockStore.sessionStorage[key] = value;`

Also, it might be easier to add this duplicate `sessionStorage` code into a `util` file and import it.

Reply:
Just created a `util` file to hold that code.